control_parameters:                     # NOTE: This must match the node name in the cpp file
  ros__parameters:
    cartesian_damping: [ 5.0, 00.0, 00.0, 00.0, 00.0, 00.0,
                        00.0,  5.0, 00.0, 00.0, 00.0, 00.0,
                        00.0, 00.0,  5.0, 00.0, 00.0, 00.0,
                        00.0, 00.0, 00.0,  1.0, 00.0, 00.0,
                        00.0, 00.0, 00.0, 00.0,  1.0, 00.0,
                        00.0, 00.0, 00.0, 00.0, 00.0,  1.0]
    cartesian_stiffness: [50.0, 00.0, 00.0, 00.0, 00.0, 00.0,
                          00.0, 50.0, 00.0, 00.0, 00.0, 00.0,
                          00.0, 00.0, 50.0, 00.0, 00.0, 00.0,
                          00.0, 00.0, 00.0,  1.0, 00.0, 00.0,
                          00.0, 00.0, 00.0, 00.0,  1.0, 00.0,
                          00.0, 00.0, 00.0, 00.0, 00.0,  1.0]
    frequency: 500.0
    joint_position_gain: 50.0
    joint_velocity_gain: 1.0
    manipulability_threshold: 0.001
